client/src/partials/templates/bhLatestInvoice.tmpl.html

```diff
@@ -0,0 +1,81 @@
+<!-- LATEST INVOICE  -->
+
+<div class="panel panel-default">
+  <div class="panel-heading">
+    <span class="glyphicon glyphicon-file"></span>
+      <strong>{{ "TABLE.COLUMNS.LAST_BILL" | translate }} </strong> 
+        <span ng-if='$ctrl.session.patientInvoice.invoicesLength'> 
+        {{ "TABLE.COLUMNS.OF" | translate }} {{ $ctrl.session.patientInvoice.invoicesLength }} {{ "TABLE.COLUMNS.INVOICES" | translate }}</span>
+  </div>
+  <div class="panel-body" style="height : 250px;">
+
+    <div style="text-align : left; font-size : 1em; ">
+      <table ng-if="$ctrl.session.patientInvoice" style="width:100%; vertical-align:top">
```

Comment:
You actually do not need `session`.  Since you are using `controllerAs`, there is no need to introduce a second object `$ctrl.session` ... you can just bind directly to `$ctrl`.